.claude/skills/commit/SKILL.md

```
@@ -4,6 +4,65 @@ description: How to write git commits in this repo that auto-close their GitHub
 
 # Commit
 
+## Conventional Commits format
+
+Every commit follows the [Conventional Commits](https://www.conventionalcommits.org)
+structure:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)